controllers/device_controller.go

```diff
@@ -286,16 +267,15 @@ func (r *DeviceReconciler) reconcileDeviceProperties(d *devicev1alpha1.Device, d
 
 		// 1.2. set the device attribute in the edge platform to the expected value
 		if desiredProperty.DesiredValue != actualProperty.ActualValue {
-			log.V(4).Info("the desired value and the actual value are different",
-				"desired value", desiredProperty.DesiredValue,
-				"actual value", actualProperty.ActualValue)
+			klog.V(4).Infof("the desired value and the actual value are different, desired: %s, actual: %s",
+				desiredProperty.DesiredValue, actualProperty.ActualValue)
 			if err := r.deviceCli.UpdatePropertyState(nil, propertyName, d, iotcli.UpdateOptions{}); err != nil {
-				log.V(4).Error(err, "failed to update property", "propertyName", propertyName)
+				klog.V(4).ErrorS(err, "failed to update property", "propertyName", propertyName)
 				failedPropertyNames = append(failedPropertyNames, propertyName)
 				continue
 			}
 
-			log.V(4).Info("successfully set the property to desired value", "property", propertyName)
+			klog.V(4).Infof("successfully set the property %s to desired value", propertyName)
```

Comment:
Adding device name into log message may be better.

Reply:
The deviceName has been added.